leader-election-impl/build.gradle

```diff
@@ -14,6 +14,8 @@ dependencies {
 
   annotationProcessor group: 'org.immutables', name: 'value'
   compileOnly 'org.immutables:value::annotations'
+  annotationProcessor 'org.derive4j:derive4j'
+  compile 'org.derive4j:derive4j-annotation'
```

Comment:
Pretty sure this needs to be `compileOnly` - `compile` would be bad because of licensing reasons